doc/developer/bindings.md

```diff
@@ -724,7 +724,7 @@ And for input parameters, the parameter may also be required:
 
 See the source documentation for each macro to read further details.  Note also
 that each possible combination of `IN`, `OUT`, and `REQ` is not
-available---output options cannot be required, and some combinations simply have
```

Comment:
This is a very intentional [em-dash](https://www.thepunctuationguide.com/em-dash.html).  I chose this representation as it's what's used in LaTeX, since this file is ASCII-encoded and not UTF-8.

Reply:
Oh ok.